lib/mocks/handlers.js

```diff
@@ -0,0 +1,20 @@
+const { rest } = require('msw');
+
+const handlers = [
+  rest.post('https://api.imgur.com/3/image', (_req, res, ctx) => {
+    const response = {
+      data: {
+        id: 'JK9ybyj',
+        deletehash: 'j83zimv4VtDA0Xp',
+        link: 'https://i.imgur.com/JK9ybyj.jpg',
+      },
+      success: true,
+      status: 200,
+    };
+    return res(ctx.json(response));
+  }),
+];
```

Comment:
mock response handler used for `uploadFile` test. As easy as adding more handlers to mock responses for other methods.